// @pnp/spfx-controls-react/lib/controls/filePicker/OneDriveFilesTab/OneDriveFilesTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./OneDriveFilesTab.module.css');
var styles = {
    filePicker: 'filePicker_b587a99d',
    tabsContainer: 'tabsContainer_b587a99d',
    tabContainer: 'tabContainer_b587a99d',
    nav: 'nav_b587a99d',
    tabHeaderContainer: 'tabHeaderContainer_b587a99d',
    breadcrumbNav: 'breadcrumbNav_b587a99d',
    breadcrumbNavItem: 'breadcrumbNavItem_b587a99d',
    tab: 'tab_b587a99d',
    tabFiles: 'tabFiles_b587a99d',
    scrollablePaneWrapper: 'scrollablePaneWrapper_b587a99d',
    actionButtonsContainer: 'actionButtonsContainer_b587a99d',
    actionButtons: 'actionButtons_b587a99d',
    actionButton: 'actionButton_b587a99d',
    tabHeader: 'tabHeader_b587a99d',
    selectedTab: 'selectedTab_b587a99d',
    header: 'header_b587a99d',
    xlg: 'xlg_b587a99d',
    xxlg: 'xxlg_b587a99d',
    xxxlg: 'xxxlg_b587a99d',
    xxxxlg: 'xxxxlg_b587a99d',
    itemPickerTopBar: 'itemPickerTopBar_b587a99d',
    commandBarNoChevron: 'commandBarNoChevron_b587a99d',
    fileItem: 'fileItem_b587a99d',
    folderItem: 'folderItem_b587a99d',
    fileTypeIcon: 'fileTypeIcon_b587a99d',
    fileTypeIconIcon: 'fileTypeIconIcon_b587a99d',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_b587a99d',
    iconColumnHeader: 'iconColumnHeader_b587a99d',
    fileRow: 'fileRow_b587a99d',
    folderRow: 'folderRow_b587a99d',
    emptyFolder: 'emptyFolder_b587a99d',
    emptyFolderImage: 'emptyFolderImage_b587a99d',
    emptyFolderImageTag: 'emptyFolderImageTag_b587a99d',
    emptyFolderTitle: 'emptyFolderTitle_b587a99d',
    emptyFolderSubText: 'emptyFolderSubText_b587a99d',
    emptyFolderMobile: 'emptyFolderMobile_b587a99d',
    emptyFolderPc: 'emptyFolderPc_b587a99d',
    emptyFolderIcon: 'emptyFolderIcon_b587a99d',
};
exports.default = styles;
/* tslint:enable */
