// @pnp/spfx-controls-react/lib/controls/filePicker/controls/DocumentTile/DocumentTile.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./DocumentTile.module.css');
var styles = {
    filePicker: 'filePicker_695c6dc2',
    tabsContainer: 'tabsContainer_695c6dc2',
    tabContainer: 'tabContainer_695c6dc2',
    nav: 'nav_695c6dc2',
    tabHeaderContainer: 'tabHeaderContainer_695c6dc2',
    breadcrumbNav: 'breadcrumbNav_695c6dc2',
    breadcrumbNavItem: 'breadcrumbNavItem_695c6dc2',
    tab: 'tab_695c6dc2',
    tabFiles: 'tabFiles_695c6dc2',
    scrollablePaneWrapper: 'scrollablePaneWrapper_695c6dc2',
    actionButtonsContainer: 'actionButtonsContainer_695c6dc2',
    actionButtons: 'actionButtons_695c6dc2',
    actionButton: 'actionButton_695c6dc2',
    tabHeader: 'tabHeader_695c6dc2',
    selectedTab: 'selectedTab_695c6dc2',
    header: 'header_695c6dc2',
    xlg: 'xlg_695c6dc2',
    xxlg: 'xxlg_695c6dc2',
    xxxlg: 'xxxlg_695c6dc2',
    xxxxlg: 'xxxxlg_695c6dc2',
    itemPickerTopBar: 'itemPickerTopBar_695c6dc2',
    commandBarNoChevron: 'commandBarNoChevron_695c6dc2',
    fileItem: 'fileItem_695c6dc2',
    folderItem: 'folderItem_695c6dc2',
    fileTypeIcon: 'fileTypeIcon_695c6dc2',
    fileTypeIconIcon: 'fileTypeIconIcon_695c6dc2',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_695c6dc2',
    iconColumnHeader: 'iconColumnHeader_695c6dc2',
    fileRow: 'fileRow_695c6dc2',
    folderRow: 'folderRow_695c6dc2',
    emptyFolder: 'emptyFolder_695c6dc2',
    emptyFolderImage: 'emptyFolderImage_695c6dc2',
    emptyFolderImageTag: 'emptyFolderImageTag_695c6dc2',
    emptyFolderTitle: 'emptyFolderTitle_695c6dc2',
    emptyFolderSubText: 'emptyFolderSubText_695c6dc2',
    emptyFolderMobile: 'emptyFolderMobile_695c6dc2',
    emptyFolderPc: 'emptyFolderPc_695c6dc2',
    emptyFolderIcon: 'emptyFolderIcon_695c6dc2',
    aboveNameplate: 'aboveNameplate_695c6dc2',
    tileContent: 'tileContent_695c6dc2',
    foreground: 'foreground_695c6dc2',
    grid: 'grid_695c6dc2',
    cellContent: 'cellContent_695c6dc2',
    content: 'content_695c6dc2',
    tile: 'tile_695c6dc2',
    invokable: 'invokable_695c6dc2',
    namePlate: 'namePlate_695c6dc2',
    nameplate: 'nameplate_695c6dc2',
    name: 'name_695c6dc2',
    isLarge: 'isLarge_695c6dc2',
    activity: 'activity_695c6dc2',
    signalField: 'signalField_695c6dc2',
    compact: 'compact_695c6dc2',
    signalFieldValue: 'signalFieldValue_695c6dc2',
    folderList: 'folderList_695c6dc2',
    link: 'link_695c6dc2',
    cell: 'cell_695c6dc2',
    listCell: 'listCell_695c6dc2',
    label: 'label_695c6dc2',
    check: 'check_695c6dc2',
    selectable: 'selectable_695c6dc2',
    showCheck: 'showCheck_695c6dc2',
    selected: 'selected_695c6dc2',
    dialogMainOverride: 'dialogMainOverride_695c6dc2',
    itemTileBlankCover: 'itemTileBlankCover_695c6dc2',
    metadata: 'metadata_695c6dc2',
    isSmall: 'isSmall_695c6dc2',
    listPage: 'listPage_695c6dc2',
    odItemTile2FolderCover: 'odItemTile2FolderCover_695c6dc2',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_695c6dc2',
    odItemTile2Image: 'odItemTile2Image_695c6dc2',
    odImageFrame2: 'odImageFrame2_695c6dc2',
    odImageFrame2Image: 'odImageFrame2Image_695c6dc2',
    odImageFrame: 'odImageFrame_695c6dc2',
    odImageStack: 'odImageStack_695c6dc2',
    odImageStackTile: 'odImageStackTile_695c6dc2',
    odImageTile: 'odImageTile_695c6dc2',
    odImageTileBackground: 'odImageTileBackground_695c6dc2',
    odImageTileImage: 'odImageTileImage_695c6dc2',
    odItemTile2SmallIcon: 'odItemTile2SmallIcon_695c6dc2',
};
exports.default = styles;
/* tslint:enable */
