// @pnp/spfx-controls-react/lib/controls/filePicker/WebSearchTab/WebSearchTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./WebSearchTab.module.css');
var styles = {
    filePicker: 'filePicker_89cdde8b',
    tabsContainer: 'tabsContainer_89cdde8b',
    tabContainer: 'tabContainer_89cdde8b',
    nav: 'nav_89cdde8b',
    tabHeaderContainer: 'tabHeaderContainer_89cdde8b',
    breadcrumbNav: 'breadcrumbNav_89cdde8b',
    breadcrumbNavItem: 'breadcrumbNavItem_89cdde8b',
    tab: 'tab_89cdde8b',
    tabFiles: 'tabFiles_89cdde8b',
    scrollablePaneWrapper: 'scrollablePaneWrapper_89cdde8b',
    actionButtonsContainer: 'actionButtonsContainer_89cdde8b',
    actionButtons: 'actionButtons_89cdde8b',
    actionButton: 'actionButton_89cdde8b',
    tabHeader: 'tabHeader_89cdde8b',
    selectedTab: 'selectedTab_89cdde8b',
    header: 'header_89cdde8b',
    xlg: 'xlg_89cdde8b',
    xxlg: 'xxlg_89cdde8b',
    xxxlg: 'xxxlg_89cdde8b',
    xxxxlg: 'xxxxlg_89cdde8b',
    itemPickerTopBar: 'itemPickerTopBar_89cdde8b',
    commandBarNoChevron: 'commandBarNoChevron_89cdde8b',
    fileItem: 'fileItem_89cdde8b',
    folderItem: 'folderItem_89cdde8b',
    fileTypeIcon: 'fileTypeIcon_89cdde8b',
    fileTypeIconIcon: 'fileTypeIconIcon_89cdde8b',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_89cdde8b',
    iconColumnHeader: 'iconColumnHeader_89cdde8b',
    fileRow: 'fileRow_89cdde8b',
    folderRow: 'folderRow_89cdde8b',
    emptyFolder: 'emptyFolder_89cdde8b',
    emptyFolderImage: 'emptyFolderImage_89cdde8b',
    emptyFolderImageTag: 'emptyFolderImageTag_89cdde8b',
    emptyFolderTitle: 'emptyFolderTitle_89cdde8b',
    emptyFolderSubText: 'emptyFolderSubText_89cdde8b',
    emptyFolderMobile: 'emptyFolderMobile_89cdde8b',
    emptyFolderPc: 'emptyFolderPc_89cdde8b',
    emptyFolderIcon: 'emptyFolderIcon_89cdde8b',
    searchBoxContainer: 'searchBoxContainer_89cdde8b',
    searchBoxMedium: 'searchBoxMedium_89cdde8b',
    searchBox: 'searchBox_89cdde8b',
    dropdownContainer: 'dropdownContainer_89cdde8b',
    filterDropdown: 'filterDropdown_89cdde8b',
    filePickerFolderCardGrid: 'filePickerFolderCardGrid_89cdde8b',
    filePickerFolderCardTile: 'filePickerFolderCardTile_89cdde8b',
    filePickerFolderCardPadder: 'filePickerFolderCardPadder_89cdde8b',
    filePickerFolderCardSizer: 'filePickerFolderCardSizer_89cdde8b',
    filePickerFolderCardImage: 'filePickerFolderCardImage_89cdde8b',
    filePickerFolderCardLabel: 'filePickerFolderCardLabel_89cdde8b',
    bingGridListCell: 'bingGridListCell_89cdde8b',
    bingTile: 'bingTile_89cdde8b',
    bingTileContent: 'bingTileContent_89cdde8b',
    bingTileFrame: 'bingTileFrame_89cdde8b',
    bingTileCheckCircle: 'bingTileCheckCircle_89cdde8b',
    bingTileNamePlate: 'bingTileNamePlate_89cdde8b',
    isSelected: 'isSelected_89cdde8b',
    copyrightLabel: 'copyrightLabel_89cdde8b',
    noResultLabel: 'noResultLabel_89cdde8b',
    bingTileThumbnail: 'bingTileThumbnail_89cdde8b',
    bingGrildList: 'bingGrildList_89cdde8b',
};
exports.default = styles;
/* tslint:enable */
