// @pnp/spfx-controls-react/lib/controls/taxonomyPicker/TaxonomyPicker.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./TaxonomyPicker.module.css');
var styles = {
    contextualMenu: 'contextualMenu_0133ff2b',
    listItem: 'listItem_0133ff2b',
    termField: 'termField_0133ff2b',
    termFieldInput: 'termFieldInput_0133ff2b',
    termFieldButton: 'termFieldButton_0133ff2b',
    termset: 'termset_0133ff2b',
    termSetSelectable: 'termSetSelectable_0133ff2b',
    termSetSelector: 'termSetSelector_0133ff2b',
    term: 'term_0133ff2b',
    termEnabled: 'termEnabled_0133ff2b',
    termDisabled: 'termDisabled_0133ff2b',
    termNoTagging: 'termNoTagging_0133ff2b',
    actions: 'actions_0133ff2b',
    termBasePicker: 'termBasePicker_0133ff2b',
    termSuggestion: 'termSuggestion_0133ff2b',
    termSuggestionSubTitle: 'termSuggestionSubTitle_0133ff2b',
    pickedTermRoot: 'pickedTermRoot_0133ff2b',
    pickedTermText: 'pickedTermText_0133ff2b',
    pickedTermCloseIcon: 'pickedTermCloseIcon_0133ff2b',
    errorMessage: 'errorMessage_0133ff2b',
    errorIcon: 'errorIcon_0133ff2b',
};
exports.default = styles;
/* tslint:enable */
