// @pnp/spfx-controls-react/lib/controls/filePicker/FilePicker.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./FilePicker.module.css');
var styles = {
    filePicker: 'filePicker_f390ec68',
    tabsContainer: 'tabsContainer_f390ec68',
    tabContainer: 'tabContainer_f390ec68',
    nav: 'nav_f390ec68',
    tabHeaderContainer: 'tabHeaderContainer_f390ec68',
    breadcrumbNav: 'breadcrumbNav_f390ec68',
    breadcrumbNavItem: 'breadcrumbNavItem_f390ec68',
    tab: 'tab_f390ec68',
    tabFiles: 'tabFiles_f390ec68',
    scrollablePaneWrapper: 'scrollablePaneWrapper_f390ec68',
    actionButtonsContainer: 'actionButtonsContainer_f390ec68',
    actionButtons: 'actionButtons_f390ec68',
    actionButton: 'actionButton_f390ec68',
    tabHeader: 'tabHeader_f390ec68',
    selectedTab: 'selectedTab_f390ec68',
    header: 'header_f390ec68',
    xlg: 'xlg_f390ec68',
    xxlg: 'xxlg_f390ec68',
    xxxlg: 'xxxlg_f390ec68',
    xxxxlg: 'xxxxlg_f390ec68',
    itemPickerTopBar: 'itemPickerTopBar_f390ec68',
    commandBarNoChevron: 'commandBarNoChevron_f390ec68',
    fileItem: 'fileItem_f390ec68',
    folderItem: 'folderItem_f390ec68',
    fileTypeIcon: 'fileTypeIcon_f390ec68',
    fileTypeIconIcon: 'fileTypeIconIcon_f390ec68',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_f390ec68',
    iconColumnHeader: 'iconColumnHeader_f390ec68',
    fileRow: 'fileRow_f390ec68',
    folderRow: 'folderRow_f390ec68',
    emptyFolder: 'emptyFolder_f390ec68',
    emptyFolderImage: 'emptyFolderImage_f390ec68',
    emptyFolderImageTag: 'emptyFolderImageTag_f390ec68',
    emptyFolderTitle: 'emptyFolderTitle_f390ec68',
    emptyFolderSubText: 'emptyFolderSubText_f390ec68',
    emptyFolderMobile: 'emptyFolderMobile_f390ec68',
    emptyFolderPc: 'emptyFolderPc_f390ec68',
    emptyFolderIcon: 'emptyFolderIcon_f390ec68',
};
exports.default = styles;
/* tslint:enable */
