// @pnp/spfx-controls-react/lib/controls/filePicker/LinkFilePickerTab/LinkFilePickerTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./LinkFilePickerTab.module.css');
var styles = {
    filePicker: 'filePicker_a9e986c1',
    tabsContainer: 'tabsContainer_a9e986c1',
    tabContainer: 'tabContainer_a9e986c1',
    nav: 'nav_a9e986c1',
    tabHeaderContainer: 'tabHeaderContainer_a9e986c1',
    breadcrumbNav: 'breadcrumbNav_a9e986c1',
    breadcrumbNavItem: 'breadcrumbNavItem_a9e986c1',
    tab: 'tab_a9e986c1',
    tabFiles: 'tabFiles_a9e986c1',
    scrollablePaneWrapper: 'scrollablePaneWrapper_a9e986c1',
    actionButtonsContainer: 'actionButtonsContainer_a9e986c1',
    actionButtons: 'actionButtons_a9e986c1',
    actionButton: 'actionButton_a9e986c1',
    tabHeader: 'tabHeader_a9e986c1',
    selectedTab: 'selectedTab_a9e986c1',
    header: 'header_a9e986c1',
    xlg: 'xlg_a9e986c1',
    xxlg: 'xxlg_a9e986c1',
    xxxlg: 'xxxlg_a9e986c1',
    xxxxlg: 'xxxxlg_a9e986c1',
    itemPickerTopBar: 'itemPickerTopBar_a9e986c1',
    commandBarNoChevron: 'commandBarNoChevron_a9e986c1',
    fileItem: 'fileItem_a9e986c1',
    folderItem: 'folderItem_a9e986c1',
    fileTypeIcon: 'fileTypeIcon_a9e986c1',
    fileTypeIconIcon: 'fileTypeIconIcon_a9e986c1',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_a9e986c1',
    iconColumnHeader: 'iconColumnHeader_a9e986c1',
    fileRow: 'fileRow_a9e986c1',
    folderRow: 'folderRow_a9e986c1',
    emptyFolder: 'emptyFolder_a9e986c1',
    emptyFolderImage: 'emptyFolderImage_a9e986c1',
    emptyFolderImageTag: 'emptyFolderImageTag_a9e986c1',
    emptyFolderTitle: 'emptyFolderTitle_a9e986c1',
    emptyFolderSubText: 'emptyFolderSubText_a9e986c1',
    emptyFolderMobile: 'emptyFolderMobile_a9e986c1',
    emptyFolderPc: 'emptyFolderPc_a9e986c1',
    emptyFolderIcon: 'emptyFolderIcon_a9e986c1',
    linkTextField: 'linkTextField_a9e986c1',
    localTabSinglePreview: 'localTabSinglePreview_a9e986c1',
    localTabSinglePreviewImage: 'localTabSinglePreviewImage_a9e986c1',
};
exports.default = styles;
/* tslint:enable */
