// @pnp/spfx-controls-react/lib/controls/filePicker/SiteFilePickerTab/SiteFilePickerTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./SiteFilePickerTab.module.css');
var styles = {
    filePicker: 'filePicker_01c2c1be',
    tabsContainer: 'tabsContainer_01c2c1be',
    tabContainer: 'tabContainer_01c2c1be',
    nav: 'nav_01c2c1be',
    tabHeaderContainer: 'tabHeaderContainer_01c2c1be',
    breadcrumbNav: 'breadcrumbNav_01c2c1be',
    breadcrumbNavItem: 'breadcrumbNavItem_01c2c1be',
    tab: 'tab_01c2c1be',
    tabFiles: 'tabFiles_01c2c1be',
    scrollablePaneWrapper: 'scrollablePaneWrapper_01c2c1be',
    actionButtonsContainer: 'actionButtonsContainer_01c2c1be',
    actionButtons: 'actionButtons_01c2c1be',
    actionButton: 'actionButton_01c2c1be',
    tabHeader: 'tabHeader_01c2c1be',
    selectedTab: 'selectedTab_01c2c1be',
    header: 'header_01c2c1be',
    xlg: 'xlg_01c2c1be',
    xxlg: 'xxlg_01c2c1be',
    xxxlg: 'xxxlg_01c2c1be',
    xxxxlg: 'xxxxlg_01c2c1be',
    itemPickerTopBar: 'itemPickerTopBar_01c2c1be',
    commandBarNoChevron: 'commandBarNoChevron_01c2c1be',
    fileItem: 'fileItem_01c2c1be',
    folderItem: 'folderItem_01c2c1be',
    fileTypeIcon: 'fileTypeIcon_01c2c1be',
    fileTypeIconIcon: 'fileTypeIconIcon_01c2c1be',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_01c2c1be',
    iconColumnHeader: 'iconColumnHeader_01c2c1be',
    fileRow: 'fileRow_01c2c1be',
    folderRow: 'folderRow_01c2c1be',
    emptyFolder: 'emptyFolder_01c2c1be',
    emptyFolderImage: 'emptyFolderImage_01c2c1be',
    emptyFolderImageTag: 'emptyFolderImageTag_01c2c1be',
    emptyFolderTitle: 'emptyFolderTitle_01c2c1be',
    emptyFolderSubText: 'emptyFolderSubText_01c2c1be',
    emptyFolderMobile: 'emptyFolderMobile_01c2c1be',
    emptyFolderPc: 'emptyFolderPc_01c2c1be',
    emptyFolderIcon: 'emptyFolderIcon_01c2c1be',
};
exports.default = styles;
/* tslint:enable */
