// @pnp/spfx-controls-react/lib/controls/richText/RichText.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./RichText.module.css');
var styles = {
    richtext: 'richtext_6c8a90e7',
    insertLinkDialog: 'insertLinkDialog_6c8a90e7',
    actions: 'actions_6c8a90e7',
    actionsRight: 'actionsRight_6c8a90e7',
    action: 'action_6c8a90e7',
    unlinkButton: 'unlinkButton_6c8a90e7',
    headerDropDown: 'headerDropDown_6c8a90e7',
    toolbarDropDown: 'toolbarDropDown_6c8a90e7',
    toolbarDropDownTitleIcon: 'toolbarDropDownTitleIcon_6c8a90e7',
    toolbarDropDownIcon: 'toolbarDropDownIcon_6c8a90e7',
    toolbarDropDownOption: 'toolbarDropDownOption_6c8a90e7',
    toolbarButtonH2: 'toolbarButtonH2_6c8a90e7',
    toolbarButtonH3: 'toolbarButtonH3_6c8a90e7',
    toolbarButtonH4: 'toolbarButtonH4_6c8a90e7',
    toolbarButtonBlockQuote: 'toolbarButtonBlockQuote_6c8a90e7',
    toolbarSubmenuCaret: 'toolbarSubmenuCaret_6c8a90e7',
    toolbarSubmenuDisplayButton: 'toolbarSubmenuDisplayButton_6c8a90e7',
};
exports.default = styles;
/* tslint:enable */
