// @pnp/spfx-controls-react/lib/controls/richText/RteColorPicker.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./RteColorPicker.module.css');
var styles = {
    ThemeColorDarker: '#004578',
    ThemeColorDark: '#005a9e',
    ThemeColorDarkAlt: '#106ebe',
    ThemeColorPrimary: '#0078d4',
    ThemeColorSecondary: '#2b88d8',
    ThemeColorTertiary: '#71afe5',
    ThemeColorNeutralSecondary: '#666666',
    ThemeColorNeutralPrimaryAlt: '#3C3C3C',
    ThemeColorNeutralPrimary: '#333333',
    ThemeColorNeutralDark: '#212121',
    colorPickerButton: 'colorPickerButton_b2d6e834',
    previewSvg: 'previewSvg_b2d6e834',
    border: 'border_b2d6e834',
    previewRectangle: 'previewRectangle_b2d6e834',
    buttonLabel: 'buttonLabel_b2d6e834',
    previewIcon: 'previewIcon_b2d6e834',
    pickerCallout: 'pickerCallout_b2d6e834',
    defaultSvg: 'defaultSvg_b2d6e834',
    fillThemeColor: 'fillThemeColor_b2d6e834',
    fillDefaultColor: 'fillDefaultColor_b2d6e834',
    pickerLabel: 'pickerLabel_b2d6e834',
    focusedContainer: 'focusedContainer_b2d6e834',
    tableRoot: 'tableRoot_b2d6e834',
    tableCell: 'tableCell_b2d6e834',
    colorCell: 'colorCell_b2d6e834',
    svg: 'svg_b2d6e834',
    selected: 'selected_b2d6e834',
};
exports.default = styles;
/* tslint:enable */
