// @pnp/spfx-controls-react/lib/controls/carousel/Carousel.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./Carousel.module.css');
var styles = {
    container: 'container_49e7f670',
    contentContainer: 'contentContainer_49e7f670',
    slideWrapper: 'slideWrapper_49e7f670',
    left: 'left_49e7f670',
    right: 'right_49e7f670',
    slideLeft: 'slideLeft_49e7f670',
    slideleft: 'slideleft_49e7f670',
    slideRight: 'slideRight_49e7f670',
    slideright: 'slideright_49e7f670',
    slideFromRight: 'slideFromRight_49e7f670',
    slidefromright: 'slidefromright_49e7f670',
    slideFromLeft: 'slideFromLeft_49e7f670',
    slidefromleft: 'slidefromleft_49e7f670',
    carouselImage: 'carouselImage_49e7f670',
    image: 'image_49e7f670',
    staticDetails: 'staticDetails_49e7f670',
    details: 'details_49e7f670',
    title: 'title_49e7f670',
    dynamicDetails: 'dynamicDetails_49e7f670',
    indicators: 'indicators_49e7f670',
    rectangle: 'rectangle_49e7f670',
    square: 'square_49e7f670',
    circle: 'circle_49e7f670',
    active: 'active_49e7f670',
    loadingComponent: 'loadingComponent_49e7f670',
    buttonLocations: 'buttonLocations_49e7f670',
    centralButtonsContainer: 'centralButtonsContainer_49e7f670',
    topButtonsContainer: 'topButtonsContainer_49e7f670',
    bottomButtonsContainer: 'bottomButtonsContainer_49e7f670',
    buttonsOnlyPrevButton: 'buttonsOnlyPrevButton_49e7f670',
    buttonsOnlyNextButton: 'buttonsOnlyNextButton_49e7f670',
    buttonsContainer: 'buttonsContainer_49e7f670',
    buttonsOnlyContainer: 'buttonsOnlyContainer_49e7f670',
    blockButtonsContainer: 'blockButtonsContainer_49e7f670',
    hiddenButtonsContainer: 'hiddenButtonsContainer_49e7f670',
};
exports.default = styles;
/* tslint:enable */
