// @pnp/spfx-controls-react/lib/controls/filePicker/StockImagesTab/StockImages.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./StockImages.module.css');
var styles = {
    filePicker: 'filePicker_47f9c83e',
    tabsContainer: 'tabsContainer_47f9c83e',
    tabContainer: 'tabContainer_47f9c83e',
    nav: 'nav_47f9c83e',
    tabHeaderContainer: 'tabHeaderContainer_47f9c83e',
    breadcrumbNav: 'breadcrumbNav_47f9c83e',
    breadcrumbNavItem: 'breadcrumbNavItem_47f9c83e',
    tab: 'tab_47f9c83e',
    tabFiles: 'tabFiles_47f9c83e',
    scrollablePaneWrapper: 'scrollablePaneWrapper_47f9c83e',
    actionButtonsContainer: 'actionButtonsContainer_47f9c83e',
    actionButtons: 'actionButtons_47f9c83e',
    actionButton: 'actionButton_47f9c83e',
    tabHeader: 'tabHeader_47f9c83e',
    selectedTab: 'selectedTab_47f9c83e',
    header: 'header_47f9c83e',
    xlg: 'xlg_47f9c83e',
    xxlg: 'xxlg_47f9c83e',
    xxxlg: 'xxxlg_47f9c83e',
    xxxxlg: 'xxxxlg_47f9c83e',
    itemPickerTopBar: 'itemPickerTopBar_47f9c83e',
    commandBarNoChevron: 'commandBarNoChevron_47f9c83e',
    fileItem: 'fileItem_47f9c83e',
    folderItem: 'folderItem_47f9c83e',
    fileTypeIcon: 'fileTypeIcon_47f9c83e',
    fileTypeIconIcon: 'fileTypeIconIcon_47f9c83e',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_47f9c83e',
    iconColumnHeader: 'iconColumnHeader_47f9c83e',
    fileRow: 'fileRow_47f9c83e',
    folderRow: 'folderRow_47f9c83e',
    emptyFolder: 'emptyFolder_47f9c83e',
    emptyFolderImage: 'emptyFolderImage_47f9c83e',
    emptyFolderImageTag: 'emptyFolderImageTag_47f9c83e',
    emptyFolderTitle: 'emptyFolderTitle_47f9c83e',
    emptyFolderSubText: 'emptyFolderSubText_47f9c83e',
    emptyFolderMobile: 'emptyFolderMobile_47f9c83e',
    emptyFolderPc: 'emptyFolderPc_47f9c83e',
    emptyFolderIcon: 'emptyFolderIcon_47f9c83e',
    stockImagesPickerContainer: 'stockImagesPickerContainer_47f9c83e',
    stockImagesPicker: 'stockImagesPicker_47f9c83e',
};
exports.default = styles;
/* tslint:enable */
