// @pnp/spfx-controls-react/lib/controls/filePicker/UploadFilePickerTab/UploadFilePickerTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./UploadFilePickerTab.module.css');
var styles = {
    filePicker: 'filePicker_ae7f25f6',
    tabsContainer: 'tabsContainer_ae7f25f6',
    tabContainer: 'tabContainer_ae7f25f6',
    nav: 'nav_ae7f25f6',
    tabHeaderContainer: 'tabHeaderContainer_ae7f25f6',
    breadcrumbNav: 'breadcrumbNav_ae7f25f6',
    breadcrumbNavItem: 'breadcrumbNavItem_ae7f25f6',
    tab: 'tab_ae7f25f6',
    tabFiles: 'tabFiles_ae7f25f6',
    scrollablePaneWrapper: 'scrollablePaneWrapper_ae7f25f6',
    actionButtonsContainer: 'actionButtonsContainer_ae7f25f6',
    actionButtons: 'actionButtons_ae7f25f6',
    actionButton: 'actionButton_ae7f25f6',
    tabHeader: 'tabHeader_ae7f25f6',
    selectedTab: 'selectedTab_ae7f25f6',
    header: 'header_ae7f25f6',
    xlg: 'xlg_ae7f25f6',
    xxlg: 'xxlg_ae7f25f6',
    xxxlg: 'xxxlg_ae7f25f6',
    xxxxlg: 'xxxxlg_ae7f25f6',
    itemPickerTopBar: 'itemPickerTopBar_ae7f25f6',
    commandBarNoChevron: 'commandBarNoChevron_ae7f25f6',
    fileItem: 'fileItem_ae7f25f6',
    folderItem: 'folderItem_ae7f25f6',
    fileTypeIcon: 'fileTypeIcon_ae7f25f6',
    fileTypeIconIcon: 'fileTypeIconIcon_ae7f25f6',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_ae7f25f6',
    iconColumnHeader: 'iconColumnHeader_ae7f25f6',
    fileRow: 'fileRow_ae7f25f6',
    folderRow: 'folderRow_ae7f25f6',
    emptyFolder: 'emptyFolder_ae7f25f6',
    emptyFolderImage: 'emptyFolderImage_ae7f25f6',
    emptyFolderImageTag: 'emptyFolderImageTag_ae7f25f6',
    emptyFolderTitle: 'emptyFolderTitle_ae7f25f6',
    emptyFolderSubText: 'emptyFolderSubText_ae7f25f6',
    emptyFolderMobile: 'emptyFolderMobile_ae7f25f6',
    emptyFolderPc: 'emptyFolderPc_ae7f25f6',
    emptyFolderIcon: 'emptyFolderIcon_ae7f25f6',
    localTabSinglePreview: 'localTabSinglePreview_ae7f25f6',
    localTabSinglePreviewImage: 'localTabSinglePreviewImage_ae7f25f6',
    localTabFilename: 'localTabFilename_ae7f25f6',
    localTabLabel: 'localTabLabel_ae7f25f6',
    localTabInput: 'localTabInput_ae7f25f6',
};
exports.default = styles;
/* tslint:enable */
