// @pnp/spfx-controls-react/lib/controls/filePicker/controls/FileBrowser/FileBrowser.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./FileBrowser.module.css');
var styles = {
    filePicker: 'filePicker_da76cb9a',
    tabsContainer: 'tabsContainer_da76cb9a',
    tabContainer: 'tabContainer_da76cb9a',
    nav: 'nav_da76cb9a',
    tabHeaderContainer: 'tabHeaderContainer_da76cb9a',
    breadcrumbNav: 'breadcrumbNav_da76cb9a',
    breadcrumbNavItem: 'breadcrumbNavItem_da76cb9a',
    tab: 'tab_da76cb9a',
    tabFiles: 'tabFiles_da76cb9a',
    scrollablePaneWrapper: 'scrollablePaneWrapper_da76cb9a',
    actionButtonsContainer: 'actionButtonsContainer_da76cb9a',
    actionButtons: 'actionButtons_da76cb9a',
    actionButton: 'actionButton_da76cb9a',
    tabHeader: 'tabHeader_da76cb9a',
    selectedTab: 'selectedTab_da76cb9a',
    header: 'header_da76cb9a',
    xlg: 'xlg_da76cb9a',
    xxlg: 'xxlg_da76cb9a',
    xxxlg: 'xxxlg_da76cb9a',
    xxxxlg: 'xxxxlg_da76cb9a',
    itemPickerTopBar: 'itemPickerTopBar_da76cb9a',
    commandBarNoChevron: 'commandBarNoChevron_da76cb9a',
    fileItem: 'fileItem_da76cb9a',
    folderItem: 'folderItem_da76cb9a',
    fileTypeIcon: 'fileTypeIcon_da76cb9a',
    fileTypeIconIcon: 'fileTypeIconIcon_da76cb9a',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_da76cb9a',
    iconColumnHeader: 'iconColumnHeader_da76cb9a',
    fileRow: 'fileRow_da76cb9a',
    folderRow: 'folderRow_da76cb9a',
    emptyFolder: 'emptyFolder_da76cb9a',
    emptyFolderImage: 'emptyFolderImage_da76cb9a',
    emptyFolderImageTag: 'emptyFolderImageTag_da76cb9a',
    emptyFolderTitle: 'emptyFolderTitle_da76cb9a',
    emptyFolderSubText: 'emptyFolderSubText_da76cb9a',
    emptyFolderMobile: 'emptyFolderMobile_da76cb9a',
    emptyFolderPc: 'emptyFolderPc_da76cb9a',
    emptyFolderIcon: 'emptyFolderIcon_da76cb9a',
};
exports.default = styles;
/* tslint:enable */
