// @pnp/spfx-controls-react/lib/controls/filePicker/controls/FolderTile/FolderTile.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./FolderTile.module.css');
var styles = {
    filePicker: 'filePicker_f1c52af5',
    tabsContainer: 'tabsContainer_f1c52af5',
    tabContainer: 'tabContainer_f1c52af5',
    nav: 'nav_f1c52af5',
    tabHeaderContainer: 'tabHeaderContainer_f1c52af5',
    breadcrumbNav: 'breadcrumbNav_f1c52af5',
    breadcrumbNavItem: 'breadcrumbNavItem_f1c52af5',
    tab: 'tab_f1c52af5',
    tabFiles: 'tabFiles_f1c52af5',
    scrollablePaneWrapper: 'scrollablePaneWrapper_f1c52af5',
    actionButtonsContainer: 'actionButtonsContainer_f1c52af5',
    actionButtons: 'actionButtons_f1c52af5',
    actionButton: 'actionButton_f1c52af5',
    tabHeader: 'tabHeader_f1c52af5',
    selectedTab: 'selectedTab_f1c52af5',
    header: 'header_f1c52af5',
    xlg: 'xlg_f1c52af5',
    xxlg: 'xxlg_f1c52af5',
    xxxlg: 'xxxlg_f1c52af5',
    xxxxlg: 'xxxxlg_f1c52af5',
    itemPickerTopBar: 'itemPickerTopBar_f1c52af5',
    commandBarNoChevron: 'commandBarNoChevron_f1c52af5',
    fileItem: 'fileItem_f1c52af5',
    folderItem: 'folderItem_f1c52af5',
    fileTypeIcon: 'fileTypeIcon_f1c52af5',
    fileTypeIconIcon: 'fileTypeIconIcon_f1c52af5',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_f1c52af5',
    iconColumnHeader: 'iconColumnHeader_f1c52af5',
    fileRow: 'fileRow_f1c52af5',
    folderRow: 'folderRow_f1c52af5',
    emptyFolder: 'emptyFolder_f1c52af5',
    emptyFolderImage: 'emptyFolderImage_f1c52af5',
    emptyFolderImageTag: 'emptyFolderImageTag_f1c52af5',
    emptyFolderTitle: 'emptyFolderTitle_f1c52af5',
    emptyFolderSubText: 'emptyFolderSubText_f1c52af5',
    emptyFolderMobile: 'emptyFolderMobile_f1c52af5',
    emptyFolderPc: 'emptyFolderPc_f1c52af5',
    emptyFolderIcon: 'emptyFolderIcon_f1c52af5',
    aboveNameplate: 'aboveNameplate_f1c52af5',
    tileContent: 'tileContent_f1c52af5',
    foreground: 'foreground_f1c52af5',
    grid: 'grid_f1c52af5',
    cellContent: 'cellContent_f1c52af5',
    content: 'content_f1c52af5',
    tile: 'tile_f1c52af5',
    invokable: 'invokable_f1c52af5',
    namePlate: 'namePlate_f1c52af5',
    nameplate: 'nameplate_f1c52af5',
    name: 'name_f1c52af5',
    isLarge: 'isLarge_f1c52af5',
    activity: 'activity_f1c52af5',
    signalField: 'signalField_f1c52af5',
    compact: 'compact_f1c52af5',
    signalFieldValue: 'signalFieldValue_f1c52af5',
    folderList: 'folderList_f1c52af5',
    link: 'link_f1c52af5',
    cell: 'cell_f1c52af5',
    listCell: 'listCell_f1c52af5',
    label: 'label_f1c52af5',
    check: 'check_f1c52af5',
    selectable: 'selectable_f1c52af5',
    showCheck: 'showCheck_f1c52af5',
    selected: 'selected_f1c52af5',
    dialogMainOverride: 'dialogMainOverride_f1c52af5',
    itemTileBlankCover: 'itemTileBlankCover_f1c52af5',
    metadata: 'metadata_f1c52af5',
    isSmall: 'isSmall_f1c52af5',
    listPage: 'listPage_f1c52af5',
    odItemTile2FolderCover: 'odItemTile2FolderCover_f1c52af5',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_f1c52af5',
    folderCover: 'folderCover_f1c52af5',
    isSmall_88d29bd4: 'isSmall_88d29bd4_f1c52af5',
    folderCoverBack: 'folderCoverBack_f1c52af5',
    folderCoverFront: 'folderCoverFront_f1c52af5',
    folderCoverFrame: 'folderCoverFrame_f1c52af5',
    folderCoverContent: 'folderCoverContent_f1c52af5',
};
exports.default = styles;
/* tslint:enable */
