// @pnp/spfx-controls-react/lib/controls/filePicker/RecentFilesTab/RecentFilesTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./RecentFilesTab.module.css');
var styles = {
    filePicker: 'filePicker_e08f9720',
    tabsContainer: 'tabsContainer_e08f9720',
    tabContainer: 'tabContainer_e08f9720',
    nav: 'nav_e08f9720',
    tabHeaderContainer: 'tabHeaderContainer_e08f9720',
    breadcrumbNav: 'breadcrumbNav_e08f9720',
    breadcrumbNavItem: 'breadcrumbNavItem_e08f9720',
    tab: 'tab_e08f9720',
    tabFiles: 'tabFiles_e08f9720',
    scrollablePaneWrapper: 'scrollablePaneWrapper_e08f9720',
    actionButtonsContainer: 'actionButtonsContainer_e08f9720',
    actionButtons: 'actionButtons_e08f9720',
    actionButton: 'actionButton_e08f9720',
    tabHeader: 'tabHeader_e08f9720',
    selectedTab: 'selectedTab_e08f9720',
    header: 'header_e08f9720',
    xlg: 'xlg_e08f9720',
    xxlg: 'xxlg_e08f9720',
    xxxlg: 'xxxlg_e08f9720',
    xxxxlg: 'xxxxlg_e08f9720',
    itemPickerTopBar: 'itemPickerTopBar_e08f9720',
    commandBarNoChevron: 'commandBarNoChevron_e08f9720',
    fileItem: 'fileItem_e08f9720',
    folderItem: 'folderItem_e08f9720',
    fileTypeIcon: 'fileTypeIcon_e08f9720',
    fileTypeIconIcon: 'fileTypeIconIcon_e08f9720',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_e08f9720',
    iconColumnHeader: 'iconColumnHeader_e08f9720',
    fileRow: 'fileRow_e08f9720',
    folderRow: 'folderRow_e08f9720',
    emptyFolder: 'emptyFolder_e08f9720',
    emptyFolderImage: 'emptyFolderImage_e08f9720',
    emptyFolderImageTag: 'emptyFolderImageTag_e08f9720',
    emptyFolderTitle: 'emptyFolderTitle_e08f9720',
    emptyFolderSubText: 'emptyFolderSubText_e08f9720',
    emptyFolderMobile: 'emptyFolderMobile_e08f9720',
    emptyFolderPc: 'emptyFolderPc_e08f9720',
    emptyFolderIcon: 'emptyFolderIcon_e08f9720',
    gridListCell: 'gridListCell_e08f9720',
    cellRight: 'cellRight_e08f9720',
    cellLastRow: 'cellLastRow_e08f9720',
    itemTile: 'itemTile_e08f9720',
    isFile: 'isFile_e08f9720',
    isPhoto: 'isPhoto_e08f9720',
    isVideo: 'isVideo_e08f9720',
    hasThumbnail: 'hasThumbnail_e08f9720',
    itemTileContent: 'itemTileContent_e08f9720',
    itemTileDiscoverFileContainer: 'itemTileDiscoverFileContainer_e08f9720',
    itemTileFileContainer: 'itemTileFileContainer_e08f9720',
    itemTileFile: 'itemTileFile_e08f9720',
    itemTileThumbnail: 'itemTileThumbnail_e08f9720',
    image: 'image_e08f9720',
    itemTileCheckCircle: 'itemTileCheckCircle_e08f9720',
    isSelected: 'isSelected_e08f9720',
    itemTileNamePlate: 'itemTileNamePlate_e08f9720',
    itemTileName: 'itemTileName_e08f9720',
    itemTileChildCount: 'itemTileChildCount_e08f9720',
    itemTileSubText: 'itemTileSubText_e08f9720',
    recentGridList: 'recentGridList_e08f9720',
};
exports.default = styles;
/* tslint:enable */
