// @pnp/spfx-controls-react/lib/controls/filePicker/controls/TilesList/TilesList.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./TilesList.module.css');
var styles = {
    filePicker: 'filePicker_258d8a27',
    tabsContainer: 'tabsContainer_258d8a27',
    tabContainer: 'tabContainer_258d8a27',
    nav: 'nav_258d8a27',
    tabHeaderContainer: 'tabHeaderContainer_258d8a27',
    breadcrumbNav: 'breadcrumbNav_258d8a27',
    breadcrumbNavItem: 'breadcrumbNavItem_258d8a27',
    tab: 'tab_258d8a27',
    tabFiles: 'tabFiles_258d8a27',
    scrollablePaneWrapper: 'scrollablePaneWrapper_258d8a27',
    actionButtonsContainer: 'actionButtonsContainer_258d8a27',
    actionButtons: 'actionButtons_258d8a27',
    actionButton: 'actionButton_258d8a27',
    tabHeader: 'tabHeader_258d8a27',
    selectedTab: 'selectedTab_258d8a27',
    header: 'header_258d8a27',
    xlg: 'xlg_258d8a27',
    xxlg: 'xxlg_258d8a27',
    xxxlg: 'xxxlg_258d8a27',
    xxxxlg: 'xxxxlg_258d8a27',
    itemPickerTopBar: 'itemPickerTopBar_258d8a27',
    commandBarNoChevron: 'commandBarNoChevron_258d8a27',
    fileItem: 'fileItem_258d8a27',
    folderItem: 'folderItem_258d8a27',
    fileTypeIcon: 'fileTypeIcon_258d8a27',
    fileTypeIconIcon: 'fileTypeIconIcon_258d8a27',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_258d8a27',
    iconColumnHeader: 'iconColumnHeader_258d8a27',
    fileRow: 'fileRow_258d8a27',
    folderRow: 'folderRow_258d8a27',
    emptyFolder: 'emptyFolder_258d8a27',
    emptyFolderImage: 'emptyFolderImage_258d8a27',
    emptyFolderImageTag: 'emptyFolderImageTag_258d8a27',
    emptyFolderTitle: 'emptyFolderTitle_258d8a27',
    emptyFolderSubText: 'emptyFolderSubText_258d8a27',
    emptyFolderMobile: 'emptyFolderMobile_258d8a27',
    emptyFolderPc: 'emptyFolderPc_258d8a27',
    emptyFolderIcon: 'emptyFolderIcon_258d8a27',
    aboveNameplate: 'aboveNameplate_258d8a27',
    tileContent: 'tileContent_258d8a27',
    foreground: 'foreground_258d8a27',
    grid: 'grid_258d8a27',
    cellContent: 'cellContent_258d8a27',
    content: 'content_258d8a27',
    tile: 'tile_258d8a27',
    invokable: 'invokable_258d8a27',
    namePlate: 'namePlate_258d8a27',
    nameplate: 'nameplate_258d8a27',
    name: 'name_258d8a27',
    isLarge: 'isLarge_258d8a27',
    activity: 'activity_258d8a27',
    signalField: 'signalField_258d8a27',
    compact: 'compact_258d8a27',
    signalFieldValue: 'signalFieldValue_258d8a27',
    folderList: 'folderList_258d8a27',
    link: 'link_258d8a27',
    cell: 'cell_258d8a27',
    listCell: 'listCell_258d8a27',
    label: 'label_258d8a27',
    check: 'check_258d8a27',
    selectable: 'selectable_258d8a27',
    showCheck: 'showCheck_258d8a27',
    selected: 'selected_258d8a27',
    dialogMainOverride: 'dialogMainOverride_258d8a27',
    itemTileBlankCover: 'itemTileBlankCover_258d8a27',
    metadata: 'metadata_258d8a27',
    isSmall: 'isSmall_258d8a27',
    listPage: 'listPage_258d8a27',
    odItemTile2FolderCover: 'odItemTile2FolderCover_258d8a27',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_258d8a27',
};
exports.default = styles;
/* tslint:enable */
